cmd/kubelet/app/server.go

```diff
@@ -783,6 +793,9 @@ func makePodSourceConfig(kc *KubeletConfig) *config.PodConfig {
 type KubeletConfig struct {
 	Address                        net.IP
 	AllowPrivileged                bool
+	Authenticator                  authenticator.Request
+	AuthorizerAttributeGetter      authorizer.RequestAttributesGetter
+	Authorizer                     authorizer.Authorizer
```

Comment:
Understood.  Stated another way, should KubeletConfig not contain a KubeletAuth, rather than these three separate fields.